iStore/Login/LoginButton.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  LoginButton.swift
+//  iStore
+//
+//  Created by Mohamed Sayed on 13.03.24.
+//
+
+import SwiftUI
+
+struct LoginButton: View {
+    
+    @EnvironmentObject var authenticationViewModel: AuthenticationViewModel
+    @EnvironmentObject var navigationCoordinator: NavigationCoordinator
```

Comment:
There is something logically wrong here. Why `LoginButton` should has var for AuthenticationViewModel and var for navigationCoordinator?

Reply:
This is just a button!

Reply:
I resolved this but I need another review.